libr/core/disasm.c

```diff
@@ -731,9 +731,17 @@ static void ds_build_op_str(RDisasmState *ds) {
 					core->parser->flagspace = -1;
 				}
 			}
+			RCore *core = ds->core;
+			ut64 p = ds->analop.ptr;
+			char *msg = calloc (sizeof (char), 128); // is this big enough?
+			r_io_read_at (core->io, p, (ut8 *)msg, 128 - 1);
```

Comment:
you can use r_io_read_i() that allows to read a value of size from 1 to 8. without having to calloc, etc

cc @condret is this function also available in siol?